src/mlpack/tests/lmnn_test.cpp

```diff
@@ -210,18 +207,18 @@ BOOST_AUTO_TEST_CASE(LMNNSeparableObjectiveTest)
 
   // Result calculated by hand.
   arma::mat coordinates = arma::eye<arma::mat>(2, 2);
-  BOOST_REQUIRE_CLOSE(lmnnfn.Evaluate(coordinates, 0, 1), 1.576, 1e-5);
-  BOOST_REQUIRE_CLOSE(lmnnfn.Evaluate(coordinates, 1, 1), 1.576, 1e-5);
-  BOOST_REQUIRE_CLOSE(lmnnfn.Evaluate(coordinates, 2, 1), 1.576, 1e-5);
-  BOOST_REQUIRE_CLOSE(lmnnfn.Evaluate(coordinates, 3, 1), 1.576, 1e-5);
-  BOOST_REQUIRE_CLOSE(lmnnfn.Evaluate(coordinates, 4, 1), 1.576, 1e-5);
-  BOOST_REQUIRE_CLOSE(lmnnfn.Evaluate(coordinates, 5, 1), 1.576, 1e-5);
+  REQUIRE(lmnnfn.Evaluate(coordinates, 0, 1) == Approx( 1.576).epsilon(1e-7));
+  REQUIRE(lmnnfn.Evaluate(coordinates, 1, 1) == Approx( 1.576).epsilon(1e-7));
+  REQUIRE(lmnnfn.Evaluate(coordinates, 2, 1) == Approx( 1.576).epsilon(1e-7));
+  REQUIRE(lmnnfn.Evaluate(coordinates, 3, 1) == Approx( 1.576).epsilon(1e-7));
+  REQUIRE(lmnnfn.Evaluate(coordinates, 4, 1) == Approx( 1.576).epsilon(1e-7));
+  REQUIRE(lmnnfn.Evaluate(coordinates, 5, 1) == Approx( 1.576).epsilon(1e-7));
```

Comment:
```suggestion
  REQUIRE(lmnnfn.Evaluate(coordinates, 0, 1) == Approx( 1.576).epsilon(1e-7));
  REQUIRE(lmnnfn.Evaluate(coordinates, 1, 1) == Approx( 1.576).epsilon(1e-7));
  REQUIRE(lmnnfn.Evaluate(coordinates, 2, 1) == Approx(1.576).epsilon(1e-7));
  REQUIRE(lmnnfn.Evaluate(coordinates, 3, 1) == Approx(1.576).epsilon(1e-7));
  REQUIRE(lmnnfn.Evaluate(coordinates, 4, 1) == Approx(1.576).epsilon(1e-7));
  REQUIRE(lmnnfn.Evaluate(coordinates, 5, 1) == Approx(1.576).epsilon(1e-7));
```

Remove the extra space.